plugins/android-documentation/build.gradle.kts

```diff
@@ -1,9 +1,22 @@
 import org.jetbrains.registerDokkaArtifactPublication
 
+plugins {
+    org.jetbrains.conventions.`kotlin-jvm`
+    org.jetbrains.conventions.`maven-publish`
+}
+
 dependencies {
+    compileOnly(project(":core"))
+    implementation(kotlin("stdlib-jdk8"))
+    implementation(kotlin("stdlib"))
+    implementation(kotlin("reflect"))
+
     implementation(project(":plugins:base"))
     testImplementation(project(":plugins:base"))
     testImplementation(project(":plugins:base:base-test-utils"))
+    testImplementation(project(":test-utils"))
+    testImplementation(project(":core:test-api"))
+    testImplementation("org.junit.jupiter:junit-jupiter:5.6.0")
 }
```

Comment:
All of the 'plugins' subprojects had these dependencies defined, because they were defined in a `subprojects {}` block

```kotlin
dependencies {
    compileOnly(project(":core"))
    implementation(kotlin("stdlib-jdk8"))
    implementation(kotlin("stdlib"))
    implementation(kotlin("reflect"))

    testImplementation(project(":test-utils"))
    testImplementation(project(":core:test-api"))
    testImplementation("org.junit.jupiter:junit-jupiter:5.6.0")
}
```

In order to keep the code working I've copy and pasted them, to recreate the same result. However, it would be nicer if these dependencies weren't so repetitive. 

Perhaps all of these dependencies aren't needed in all 'plugins' projects and some can be removed?

Alternatively, perhaps these dependencies could be defined as API dependencies in another project (like `:plugins:base`?)

```kotlin
dependencies {
    compileOnlyApi(project(":core"))
    api(kotlin("stdlib-jdk8"))
    api(kotlin("stdlib"))
    api(kotlin("reflect"))
    testApi(project(":test-utils"))
    testApi(project(":core:test-api"))
    testApi("org.junit.jupiter:junit-jupiter:5.6.0")
}
```

then each plugin project would automatically get these dependencies.

Reply:
Yes I agree - postpone trying to optimise the dependencies.